src/pages/PlayVideo/PlayVideo.jsx

```diff
@@ -0,0 +1,174 @@
+    <div
+      className={`content flex-row-wrap flex-mid-center default ${styles.container}`}
+    >
+      <div
+        className={`${styles.videoSection} flex-column-wrap flex-mid-center`}
+      >
+        <div className={styles.player}>
+          <ReactPlayer
+            url={`http://www.youtube.com/watch?v=${videoID}`}
+            controls={true}
+            className={styles.video}
+            width="100%"
+            height="100%"
+          />
+        </div>
+
+        <div
+          className={`flex-row-wrap flex-mid-left ${styles.option_container} `}
+        >
+          {" "}
+          <div
+            className={`flex-column-wrap bottom_border ${styles.title_container}`}
+          >
+            <p className="header-6">{videoData?.title}</p>
+            <p className="body-2 grey_text">{videoData?.views} Views</p>
+          </div>
+          <div
+            className={`flex-row-wrap bottom_border  ${styles.btn_container}`}
+          >
+            <div
+              onClick={() => likeHandler(videoData, userDispatch)}
+              className="flex-column-wrap flex-mid-center"
+            >
+              <button
+                className={`btn btn_action  ${
+                  userState.liked?.includes(videoData) ? styles.active : ""
+                }`}
+              >
+                <span className={`material-icons`}>thumb_up</span>
+              </button>
+              <p className="subtitle-1">Like</p>
+            </div>
+            <div
+              onClick={() => dislikeHandler(videoData)}
+              className="flex-column-wrap flex-mid-center"
+            >
+              <button
+                className={`btn btn_action  ${dislike ? styles.active : ""}`}
+              >
+                <span className={`material-icons`}>thumb_down</span>
+              </button>
+              <p className="subtitle-1">Dislike</p>
+            </div>
+            <div
+              onClick={() =>
+                watchlaterHandler(videoData)
+                  ? removeFromWatchlater(videoData, userDispatch)
+                  : addToWatchlater(videoData, userDispatch)
+              }
+              className="flex-column-wrap flex-mid-center"
+            >
+              <button
+                className={`btn btn_action ${
+                  watchlaterHandler(videoData) ? styles.active : ""
+                }  `}
+              >
+                <span className={`material-icons`}>watch_later</span>
+              </button>
+              <p className="subtitle-1">Watch Later</p>
+            </div>
+            {/* <div
+              onClick={() => addToWatchlater(videoData, videoDispatch)}
+              className="flex-column-wrap flex-mid-center"
+            >
+              <button className={`btn btn_action  `}>
+                <span className={`material-icons`}>playlist_add</span>
+              </button>
+              <p className="subtitle-1">Watch Later</p>
+            </div> */}
```

Comment:
You can remove comments if it's no longer needed.

Reply:
Yes, thank You. Will be removing them.